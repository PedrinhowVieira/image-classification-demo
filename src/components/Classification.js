import React from "react";

const classification = () => (
  <img src={require('./../Images/gallery2.jpeg')} alt=""/>
);

export default classification;
