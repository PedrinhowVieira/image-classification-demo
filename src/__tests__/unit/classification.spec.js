import React from 'react';
import renderer from 'react-test-renderer';

import Classification from '../../components/Classification';

describe('<Classification />', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Classification />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
