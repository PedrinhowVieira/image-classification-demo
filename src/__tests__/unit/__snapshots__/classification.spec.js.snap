// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Classification /> renders an <img /> component 1`] = `
<img
  alt=""
  src="gallery2.jpeg"
/>
`;

exports[`<Classification /> renders correctly 1`] = `
<img
  alt=""
  src="gallery2.jpeg"
/>
`;
